pkgs/by-name/ru/rust-streamdeck/package.nix

```diff
@@ -0,0 +1,50 @@
+{
+  rustPlatform,
+  fetchFromGitHub,
+  pkg-config,
+  lib,
+  udev,
+  nix-update-script,
+  versionCheckHook,
+}:
+rustPlatform.buildRustPackage rec {
+  pname = "rust-streamdeck";
+  version = "0.9.0";
+
+  src = fetchFromGitHub {
+    owner = "ryankurte";
+    repo = "rust-streamdeck";
+    rev = "v${version}";
+    hash = "sha256-9FuTnRQHKYJzMqhhgyTVq2R+drn4HAr3GDNjQgc3r+w=";
+  };
+
+  postPatch = ''
+    ln -s ${./Cargo.lock} Cargo.lock
+  '';
+
+  cargoLock = {
+    lockFile = ./Cargo.lock;
```

Comment:
Please ask upstream to include a Cargo.lock. We try to avoid including lockfiles in nixpkgs.

Reply:
Done here: https://github.com/ryankurte/rust-streamdeck/pull/38

Reply:
done!